lib/codebreaker/player.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module Codebreaker
+  # Player entity class
+  class Player
+    attr_accessor :attempts_total, :hints_total
+    attr_reader :name, :difficulty, :attempts_used, :hints_used
+
+    def initialize(name)
+      @name = name
+      @attempts_total = 0
+      @hints_total = 0
+    end
+
+    def current_difficulty(difficulty)
+      @difficulty = difficulty
+    end
+
+    def attempts_used=(game_attempt)
```

Comment:
It's weird method. It's expected that methods: `var_name = (value)` stores value to `var_name`, but u have additional logic here. Solution can be: move this logic to method call or changing method to something like `update_stats` with alll logic